src/Veldrid/OpenGL/OpenGLPipeline.cs

```diff
@@ -364,7 +377,31 @@ void ReportInvalidResourceName(string resourceName)
                 uniformIndex++;
             }
 
-            throw new VeldridException($"Unable to bind uniform \"{resourceName}\" by name. Valid names for this pipeline are: {string.Join(", ", names)}");
+            throw new VeldridException($"Unable to bind uniform \"{uniformName}\" by name. Valid names for this pipeline are: {string.Join(", ", names)}");
+        }
+
+        void ReportInvalidResourceName(string resourceName, ProgramInterface resourceType)
+        {
+            int maxLength = 0;
+            int resourceCount = 0;
+            glGetProgramInterfaceiv(_program, resourceType, ProgramInterfaceParameterName.MaxNameLength, &maxLength);
```

Comment:
These need to be hidden behind a capability/version check, because they are only core in version 4.3+, even if they are behind the DEBUG and GL_VALIDATE_SHADER_RESOURCE_NAMES flags

Reply:
@mellinoe 
Updated - is this the preferred pattern for version checks?

Reply:
It would be preferable to provide a fallback for version that don't support this particular functionality. Is that feasible in this case?